language_formatters_pre_commit_hooks/pretty_format_yaml.py

```diff
@@ -14,6 +15,28 @@
 from six import text_type
 
 
+def process_single_document(document, yaml):
+    """Pretty format one document containing YAML or primitive (non-YAML) text.
+    Call this function for each doc within a multi-document .yaml file.
+
+    Args:
+        document (str): Original document content.
+        yaml: YAML library instance.
+
+    Returns:
+        Pretty-formatted content (str).
+    """
```

Comment:
Small rewording, mostly needed to generalise the documentation as the task done by the method is pretty general

```suggestion
    """Pretty format one YAML document.
    
    This is needed in order to prevent `ruamel.yaml` to interfere with documents that have primitive types on the document root.
     For more context check https://github.com/macisamuele/language-formatters-pre-commit-hooks/pull/1

    Args:
        document (str): Original document content.
        yaml: YAML library instance.
    Returns:
        Pretty-formatted content (str).
    """
```